vcpkg/readme/vcpkg-README.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Microsoft/vcpkg README
+description: README for vcpkg registry
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 10/17/2023
+ms.prod: vcpkg
+ROBOTS: NOINDEX
+---
+
+# vcpkg: Overview
+
+vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.
+This tool and ecosystem are constantly evolving, and we always appreciate contributions!
+
+If you've never used vcpkg before, or if you're trying to figure out how to use vcpkg,
+check out our [guide to start using vcpkg](../get_started/get-started.md).
```

Comment:
This sentence feels superfluous given that there is a "Getting Started" section later that requests the user go to the same tutorial.